### DISCUSSION TITLE
Help: Getting data from batteries but nothing showing up on HA... mttq problems?

### DISCUSSION THREAD

**Discussion by rdstevens** [fl4p/batmon-ha#92] [General]
I'm a complete noob to this... but I'm a developer and prepared to spend some time working through problems (and documenting resolutions).

I think I need some help: I'm getting data from the 3 BMSes (according to the logs), but nothing is showing up in HA.

Can anyone give me a hint?
Thanks,
Rob

**Comment by rdstevens** [comment]
The battery alias should not have spaces in its name.

**Comment by fl4p** [comment]
I'll fix this with the next release
